Builds/containers/shared/rippled.service

```diff
@@ -1,12 +1,12 @@
 [Unit]
 Description=Ripple Daemon
+After=network-online.target
+Wants=network-online.target
 
 [Service]
 Type=simple
 ExecStart=/opt/ripple/bin/rippled --net --silent --conf /etc/opt/ripple/rippled.cfg
-# Default KillSignal can be used if/when rippled handles SIGTERM
-KillSignal=SIGINT
-Restart=no
+Restart=yes
```

Comment:
What is the intention of this line? Based on the documentation of systemd.service, it did not list =yes as an option. There are various configuration we can pass to Restart= depending on what we want. do we want to restart the service even if it has a clean exit code initiated by an operator? I used this link as my reference http://manpages.ubuntu.com/manpages/cosmic/man5/systemd.service.5.html

Reply:
Yeah, sorry. Brain shortcircuit, this should be `on-failure`